---
layout: page
title: " Systematic mapping of occluded genes by cell fusion reveals prevalence and stability of cis-mediated silencing in somatic cells."
breadcrumb: true
categories:
    - publication
## publication related information
pub:
    authors: " Timothy J. Looney, Li Zhang, Chih-Hsin Chen, Jae Hyun Lee, Sheila Chari, Frank Fuxiang Mao, Mattia Pelizzola, Lu Zhang, Ryan Lister, Samuel W. Baker, Croydon J. Fernandes, Jedidiah Gaetz, Kara M. Foshay, Kayla L. Clift, Zhenyu Zhang, Wei-Qiang Li, Eric J. Vallender, Ulrich Wagner, Jane Yuxia Qin, Katelyn J. Michelini, Branimir Bugarija, Donghyun Park, Emmanuel Aryee, Thomas Stricker, Jie Zhou, Kevin P. White, Bing Ren, Gary P. Schroth, Joseph R. Ecker, Andy Peng Xiang,  Bruce T. Lahn"
    journal: " Genome research"
    date: 2014-02
    doi:  10.1101/gr.143891.112
    volume:  24
    pages:  267--280
    number:  2
    abstract: " Both diffusible factors acting in trans and chromatin components acting in cis are implicated in gene regulation, but the extent to which either process causally determines a cell's transcriptional identity is unclear. We recently used cell fusion to define a class of silent genes termed cis-silenced (or occluded) genes, which remain silent even in the presence of trans-acting transcriptional activators. We further showed that occlusion of lineage-inappropriate genes plays a critical role in maintaining the transcriptional identities of somatic cells. Here, we present, for the first time, a comprehensive map of occluded genes in somatic cells. Specifically, we mapped occluded genes in mouse fibroblasts via fusion to a dozen different rat cell types followed by whole-transcriptome profiling. We found that occluded genes are highly prevalent and stable in somatic cells, representing a sizeable fraction of silent genes. Occluded genes are also highly enriched for important developmental regulators of alternative lineages, consistent with the role of occlusion in safeguarding cell identities. Alongside this map, we also present whole-genome maps of DNA methylation and eight other chromatin marks. These maps  uncover a complex relationship between chromatin state and occlusion. Furthermore, we found that DNA methylation functions as the memory of occlusion in a subset of occluded genes, while histone deacetylation contributes to the implementation but not memory of occlusion. Our data suggest that the identities  of individual cell types are defined largely by the occlusion status of their genomes. The comprehensive reference maps reported here provide the foundation for future studies aimed at understanding the role of occlusion in development and disease.,"
---